tsconfig.json

```diff
@@ -50,6 +50,6 @@
   "angularCompilerOptions": {
     "fullTemplateTypeCheck": true,
     "strictInjectionParameters": true,
-    "enableIvy": false
+    "enableIvy": true
```

Comment:
Angular team does not yet recommend enabling this ( https://angular.io/guide/creating-libraries#publishing-your-library ). What is the reason behind this decision?

Reply:
That recommendation is for libraries written in angular 9 where ivy was optional.  

I was thinking that if you are using angular 9 and the ng-material-extensions library then you are going to be consuming a 9.0.x library version and that if you are using angular 10 then you would use the 10.0.x version (this pull request) so that that guidance would not apply here.

My angular 10 project using ng-material-extensions version 9 doesn't build right with aot on.  I have to test it with this pull request build to see if it works and as part of that I can see if this switch is important or not.